index.html

```diff
@@ -8,21 +8,30 @@
     <link rel="stylesheet" href="main.css">
 </head>
 <body>
-    <div class="wrapper">
             <header>
                 <nav class="nav-bar">
                     <a href="#">
                         <div class="logo">My Logo</div>
-                        <img src="images/Enabled.png" alt="enable-image">
-                      </a>
-                      <ul class="desktop-nav">
+                      </a> 
+                     
+                    <div>
+                        <img class="loose" src="images/Enabled.png" alt="enable-image">
+                    </div>
+                        <ul class="pop-up">
+                            <li><a href="#portfolio"class="nav-links" >Portfolio</a></li>
+                            <li><a href="#about"class="nav-links" >About</a></li>
+                            <li><a href="#contact" class="nav-links" >Contact</a></li>
+                          </ul>
```

Comment:
- [ ] Kindly check your design on the popup menu, the background should be blurred but not entirely blue as in your case check the comparison below;

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/173593004-95050a7d-e370-4b6b-89e5-bae37b4b2bbf.png) | ![image](https://user-images.githubusercontent.com/61408860/173593113-5da468d3-c649-4d00-979d-abab8efa1a65.png)